docs/components/popover/test.html

```diff
@@ -8,7 +8,39 @@
 
 {% block content %}
   <section>
-    <h2>Appearance</h2>
+    <header>
+      <h2 id="button-popover-open-page-load">Button with Popover Open on Page Load</h2>
+    </header>
+    <hx-div class="hxLg">
+      <button class="hxBtn" id="openButtonPopover">
+        Toggle
+      </button>
+      <hx-popover
+        id="demoOpenPopoverTest"
```

Comment:
not necessary, but can alphabetize block

Reply:
block alphabetized